games/static/games/proto.js

```diff
@@ -4,31 +4,56 @@ class Game {
     this.setup = setup;
     this.grid = document.querySelector(".grid");
     this.startButton = document.querySelector("#start");
+    this.restartButton = document.querySelector("#restart");
     this.isRunning = false
   }
   start() {
-    let alienMovementId = 0;
+    let gameId = 0;
+    let cleanId = 0;
+    let start
```

Comment:
mmm this is also wierd... in js you should have to each definition left hand side and right hand side (LHS & RHS).
so you need to initialize `start` and `restart`.
also - those vars are not properly named